proto/tripcommon.proto

```diff
@@ -95,6 +95,7 @@ message Location {
   optional uint32 waypoint_index = 31;
   optional SearchFilter search_filter = 32;
   optional uint32 street_side_max_distance = 33;
+  optional int32 preferred_z_level = 34;
```

Comment:
any chance we could call this `preferred_layer` so that it closely matches the osm tagging name?

Reply:
i guess we probably should have named it layer in edgeinfo and TaggedValue as well then?

Reply:
It is actually `z_level` there:
https://github.com/valhalla/valhalla/blob/9500f40838f30a2cd8ddc81c19a9ffdc6b744591/valhalla/baldr/edgeinfo.h#L195

We can do as you wish:
1. Rename all of them to `layer`(I don't think it should be even called as breaking change since we merged it 2 days ago)
2. Rename only this field
3. Leave it as is

WDYT?

Reply:
I addressed all other review notes, let me know your decision here and I'll finish it.

Reply:
i would love to rename all of them, option 1

Reply:
Okay, will do 👍

Reply:
Done, please take a look one more time @kevinkreiser